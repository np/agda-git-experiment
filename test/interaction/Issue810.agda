-- {-# OPTIONS -v interaction:100 #-}
module Issue810 where

postulate
  T : Set → Set

introHid : {A : Set} → T A
introHid = {!!}

data Sg {A : Set} : A → Set where
  sg : (a : A) → Sg a

intro : ∀ {A}{a : A} → Sg a
intro = {!!}

intro′ : ∀ {A}(a : A) → Sg a
intro′ = {!!}
