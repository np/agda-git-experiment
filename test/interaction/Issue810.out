Agda2> (agda2-status-action "")
(agda2-info-action "*Type-checking*" "" nil)
(agda2-highlight-clear)
(agda2-info-action "*Type-checking*" "Checking Issue810 (Issue810.agda).\n" t)
(agda2-info-action "*Type-checking*" "Finished Issue810.\n" t)
(agda2-status-action "")
(agda2-info-action "*All Goals*" "?0 : {A : Set} → T A\n?1 : {A : Set} {a : A} → Sg a\n?2 : {A : Set} (a : A) → Sg a\n" nil)
((last . 1) . (agda2-goals-action '(0 1 2)))
Agda2> (agda2-give-action 0 "λ {A} → ?")
(agda2-status-action "")
(agda2-info-action "*All Goals*" "?1 : {A : Set} {a : A} → Sg a\n?2 : {A : Set} (a : A) → Sg a\n?3 : T A\n" nil)
((last . 1) . (agda2-goals-action '(3 1 2)))
Agda2> (agda2-give-action 1 "sg ?")
(agda2-status-action "")
(agda2-info-action "*All Goals*" "?2 : {A : Set} (a : A) → Sg a\n?3 : T A\n?4 : .A\n" nil)
((last . 1) . (agda2-goals-action '(3 4 2)))
Agda2> (agda2-give-action 2 "λ a → ?")
(agda2-status-action "")
(agda2-info-action "*All Goals*" "?3 : T A\n?4 : .A\n?5 : Sg a\n" nil)
((last . 1) . (agda2-goals-action '(3 4 5)))
Agda2> 